backend/src/main/java/com/woowacourse/levellog/application/TeamService.java

```diff
@@ -96,19 +95,19 @@ private Long getHostId(final List<Participant> participants) {
                 .getId();
     }
 
-    private ParticipantsResponse getParticipantsResponse(final List<Participant> participants) {
-        final List<ParticipantResponse> participantResponses = participants.stream()
+    private List<ParticipantResponse> getParticipantsResponse(final List<Participant> participants) {
+        return participants.stream()
                 .map(it -> new ParticipantResponse(
                         it.getId(),
```

Comment:
```suggestion
                        it.getMember().getId(),
```
가 되야 될 것 같습니다. 현재는 participant의 id가 넣어지는 것 같습니다.

Reply:
API 명세서에서 `participants`의 `id`가 Participant도메인의 id가 아닌, member의 아이디였군요..!
다른 곳은 모두 도메인의 `id`를 사용해서 헷갈렸네요🥲 수정하고, 명세서에도 기록해 놓겠습니다〰️
```json
{
  "teams" : [
  {
	  "id" : 1,
	  "title": "제이슨조",
	  "place": "잠실 트랙룸",
	  "startAt": "2022-07-15",
	  "teamImage": "http://image.com",
	  "hostId": 1,
	  "participants": [{
		  "id" : 1,
		  "levellogId" : null,
		  "nickname" : "페퍼",
		  "profileUrl" : "https://avatars.githubusercontent.com/u/28749734?v=4"
	  }, {
		  "id" : 2,
		  "levellogId" : 6,
		  "nickname" : "OzRagwort",
		  "profileUrl" : "https://avatars.githubusercontent.com/u/32123302?v=4"
	  }, {
		  "id" : 3,
		  "levellogId" : 4,
		  "nickname" : "유정이",
		  "profileUrl" : "https://avatars.githubusercontent.com/u/76840965?v=4"
	  }]
  }
  ]
}
```